### PR TITLE
Add test for duplicate inserts & redundant deletes

### PR DESCRIPTION
**Pull request by krishnangovindraj**
## Usage and product changes
Add test for duplicate inserts & redundant deletes

## Implementation
Add test for duplicate inserts & redundant deletes

### CODE REVIEW

File: query/language/delete.feature [hunk 1]
```diff
@@ -792,6 +792,51 @@ Feature: TypeQL Delete Query
...
+       """
+    Given transaction commits
+
+    Given connection open write transaction for database: typedb
+    When get answers of typeql read query
+    """
+    match
+      $e isa internship, links (intern: $p);
+    """
+    Then answer size is: 1
```
Comment (farost) [inline query/language/delete.feature:821]: It could be read easier if we actually checked what are the answers. For example, their labels. At least it's easier for me when I skim old (even my) tests

File: query/language/insert.feature [hunk 2]
```diff
@@ -208,6 +208,17 @@ Feature: TypeQL Insert Query
       insert $x isa factory;
       """
 
+
+  Scenario: A single variable may not have multiple isa constraints in an insert stage
+    When typeql write query; fails with a message containing: "Found multiple insert statements for the variable"
```
Comment (farost) [inline query/language/insert.feature:213]: Not sure if `multiple insert statements` is enough for a user to identify that the problem is in `isa`s! But it might be reviewed in the server's PR, seeing the whole stack trace. It should probably highlight the code region causing the error

**Review by farost — APPROVED** [review]

File: query/language/delete.feature [hunk 3]
```diff
@@ -1430,6 +1466,8 @@ Feature: TypeQL Delete Query
     Then answer size is: 0
 
 
+  # TODO: 3.x: Needs finer write type-checks
+  @ignore
```
Comment (krishnangovindraj) [inline query/language/delete.feature:1470]: This is quite bad.
